extensions/wikia/UserRenameTool/UserRenameToolHelper.class.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+class UserRenameToolHelper {
+	const COMMUNITY_CENTRAL_CITY_ID = 177;
+
+	public static function getCentralUserTable() {
+		return F::app()->wg->ExternalSharedDB . '.user';
+	}
+
+	/**
+	 * Gets wikis an IP address might have edits on
+	 *
+	 * @param String $ipAddress The IP address to lookup
+	 * @return array
+	 */
+	public static function lookupIPActivity( $ipAddress ) {
+		global $wgDevelEnvironment, $wgSpecialsDB;
+
+		if ( empty( $ipAddress ) || !IP::isIPAddress( $ipAddress ) ) {
+			return false;
+		}
+
+		$result = [];
+		$ipLong = ip2long( $ipAddress );
+
+		if ( empty( $wgDevelEnvironment ) ) {
+			$dbr = wfGetDB( DB_SLAVE, [], $wgSpecialsDB );
+			$res = $dbr->select(
+				[ 'multilookup' ],
+				[ 'ml_city_id' ],
+				[
+					'ml_ip' => $ipLong,
+				],
+				__METHOD__
+			);
+
+			/** @var Object $row */
+			foreach ( $res as $row ) {
+				if ( WikiFactory::isPublic( $row->ml_city_id ) ) {
+					$result[] = (int)$row->ml_city_id;
+				}
+			}
+
+			$dbr->freeResult( $res );
+		} else { // on devbox - set up the list manually
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)